lib/l10n/app_bg.arb

```diff
@@ -0,0 +1,6 @@
+{
+  "appName": "Help A Paw",
```

Comment:
Hey, sorry about misunderstanding - I mean we can hardcode the name as it's not going to change in any language.

Reply:
Oh okay copy that so i will leave the .arb empty for now ?